samples/matter/light_bulb/child_image/mcuboot_single_image_smp_dfu.conf

```diff
@@ -9,5 +9,21 @@ CONFIG_NORDIC_QSPI_NOR=y
 CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
 CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
 
-CONFIG_MULTITHREADING=y
+CONFIG_BOOT_UPGRADE_ONLY=y
```

Comment:
Ok, now I remember why it was added only for nRF53. That was to provide the situation when net core image is applied, but app core image is reverted, so we end up with two not compatible core images and application crashes, so update is not possible anymore. I think we should not enable it for nRF52, as rollback mechanism is important for users.

Reply:
@ArekBalysNordic noooo :(

Reply:
If it is set to y we need about 150 B more to optimize to fit in 24k, ehh

Reply:
Ok, we can discuss that on Monday within wider audience, but I think it may be a better idea to use 32kB partition than remove the rollback mechanism.

Reply:
Rollback mechanism is a must for us. I agree with Damians suggestion.